cmd/ltapi/main.go

```diff
@@ -18,6 +18,7 @@ import (
 func RunServerCmdF(cmd *cobra.Command, args []string) error {
 	port, _ := cmd.Flags().GetInt("port")
 
+	// TODO: add a config file for the API server.
```

Comment:
This is getting really out of hand now with so many config files :)

Reply:
Well, I am still not convinced by the monolithic config approach :)